src/tar_check.rs

```diff
@@ -111,17 +111,26 @@ fn tar_check_archive<R: Read>(mut archive: Archive<R>, path_str: &str) {
 		if suid_files.is_empty() {
 			eprintln!("Package {} has no SUID files.", path_str);
 		}
+		eprint!("{}{}, ", "[E]".bold(), "=list executable files");
+		eprint!("{}{}, ", "[L]".bold(), "=list all files");
+		eprint!("{}{}, ", "[T]".bold(), "=run shell to inspect");
```

Comment:
This would give a compiler warning

```
error: literal with an empty format string
   --> src/tar_check.rs:114:35
    |
114 |         eprint!("{}{}, ", "[E]".bold(), "=list executable files");
    |                                         ^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D clippy::print-literal` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#print_literal
help: try this
    |
114 -         eprint!("{}{}, ", "[E]".bold(), "=list executable files");
114 +         eprint!("{}=list executable files, ", "[E]".bold());
    | 

```

I'll revert to inlining the "static" text as the compiler suggests, but will keep the `.bold()` part you've introduced